curations/nuget/nuget/-/System.Data.DataSetExtensions.yaml

```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: System.Data.DataSetExtensions
+  provider: nuget
+  type: nuget
+revisions:
+  4.5.0:
+    licensed:
+      declared: MIT
```

Comment:
https://www.nuget.org/packages/System.Data.DataSetExtensions links to https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT

Reply:
I followed a process like this for the rest of this items.  Please ask me if you have questions